pkg/pkg.mk

```diff
@@ -31,9 +31,9 @@ ifeq (1,$(PKG_BUILD_OUT_OF_SOURCE))
   PKG_SOURCE_DIR ?= $(PKGDIRBASE)/$(PKG_NAME)
```

Comment:
I think there's an issue here. on murdock, they's be the same, as PKGDIRBASE is set to $(BINDIR)/pkg.
That can be changed, should we change that to "$(BINDIR)/pkg-src"?

Reply:
I mean, change in .murdock?
alternatively, rename this PKG_BUILD_DIR here to .../pkg-build.

Reply:
I think it would be better if wouldn't have to modify Murdock config. I'll adapt.

Reply:
I opted for the pkg-build version.